src/app/components/Button.tsx

```diff
@@ -109,8 +109,12 @@ export const PlainButton = styled.button`
 // tslint:disable-next-line:variable-name
 export const ButtonLink = styled(PlainButton)`
   outline: none;
-  ${textStyle}
-  ${(props: {decorated: boolean}) => props.decorated ? decoratedLinkStyle : linkStyle}
+  ${(props: {decorated: boolean, disabled: boolean}) =>
+    props.disabled ? `
+      cursor: not-allowed;
```

Comment:
are you sure you want to remove the link styling? is there design for this? i would think that you'd leave the existing styling and just add the disabled cursor.

you should do the cursor in a separate `${}` of the `ButtonLink` styles, there is no reason to make this logic so crazy.